mut/redirects/redirect_main.py

```diff
@@ -121,8 +122,14 @@ def parse_source_file(source_path: str, output: str) -> None:
                 if keyword_split[0] == 'symlink':
                     type_split = line.split(':', 1)
                     sym_split = [sym.strip() for sym in type_split[1].split('->')]
-                    sym_tup = (sym_split[0], sym_split[1])
-                    rc.symlinks.append(sym_tup)
+                    alias = sym_split[0]  # did not exist previously
+                    origin = sym_split[1]  # original folder
+
+                    if os.path.islink('./build/public/' + alias):
+                        os.remove('./build/public/' + alias)
+
+                    os.symlink('./build/public/' + origin, './build/public/' + alias)
```

Comment:
So... this is a fun part of symlinks. You actually want 
```python
os.symlink(os.path.join('./build/public/', origin), alias)
```

Because the alias is relative to the path of the symlink, not where you currently are.